src/nyc_trees/apps/census_admin/views.py

```diff
@@ -3,33 +3,71 @@
 from __future__ import unicode_literals
 from __future__ import division
 
+from collections import OrderedDict
+import json
 
-def start_admin_users_job(request):
-    # TODO: implement
-    # TODO: implement
-    pass
+from django.contrib.gis.geos import MultiPolygon, Polygon
+from django.db import transaction
 
+from apps.core.models import Group
 
-def start_admin_surveys_job(request):
-    # TODO: implement
-    pass
 
+def upload_group_polygons(request):
+    file_obj = request.FILES['file']
+    file_obj.seek(0)
+    text = file_obj.read()
+    data = json.loads(text)
```

Comment:
I just saw that we already report other types of errors, so let's do that instead of raising 400.